arm-sql/2014-04-01/swagger/importExport.json

```diff
@@ -0,0 +1,533 @@
+                        "format": "uuid",
+                        "description": "The operation ID associated with the operation."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/ImportExportOperationResponse"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Get Import/Export operations by server": { "$ref": "../examples/ImportExportOperationGetByServer.json" }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/importExportOperationResults/{operationId}": {
+            "get": {
+                "tags": [
+                    "ImportExport"
+                ],
+                "operationId": "ImportExportOperations_GetByDatabase",
+                "description": "The status of an import or export operation on a database given the operation ID.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameter"
+                    },
+                    {
+                        "name": "databaseName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the database"
+                    },
+                    {
+                        "name": "operationId",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "format": "uuid",
+                        "description": "The operation ID associated with the operation."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/ImportExportOperationResponse"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Get Import/Export operations by database": { "$ref": "../examples/ImportExportOperationGetByDatabase.json" }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "ImportExtensionProperties": {
+            "properties": {
+                "operationMode": {
```

Comment:
As this is always import, it is better to remove it in next API version. Please log a bug against the service.